src/distrs/beta.jl

```diff
@@ -1,7 +1,6 @@
 # functions related to beta distributions
 
 import .RFunctions:
-    betalogpdf,
```

Comment:
Yes you are right. I guess in order to use the original `Rmath` functions for `Float64` and my code for other generic types, I also need to change the original definition of the imports in`rmath.jl` from `Real` to `Float64`. Is my understanding right?